src/lava/utils/weightutils.py

```diff
@@ -17,81 +18,154 @@ class SignMode(Enum):
     INHIBITORY = 3
 
 
-def optimize_weight_bits(weight: np.ndarray,
-                         loihi2: bool = False) -> ty.Tuple[np.ndarray,
-                                                           int,
-                                                           int,
-                                                           SignMode]:
+def determine_sign_mode(weights: np.ndarray) -> SignMode:
```

Comment:
Please add a docstring

Reply:
Addressed in https://github.com/lava-nc/lava/pull/328/commits/7476abe0ebe64dd4fb79c079d553955a2f4f94fd